src/main/kotlin/com/linchpino/core/LinchpinApp.kt

```diff
@@ -9,5 +9,5 @@ import org.springframework.data.jpa.repository.config.EnableJpaAuditing
 class LinchpinApp
 
 fun main(args: Array<String>) {
-    runApplication<LinchpinApp>(*args)
+	runApplication<LinchpinApp>(*args)
```

Comment:
Please add a ".editorconfig" file to the root of the project to have the same formatting for the whole project.

Reply:
Can you execute the formatter for the whole project?